app/assets/javascripts/admin/line_items/controllers/line_items_controller.js.coffee

```diff
@@ -104,15 +104,38 @@ angular.module("admin.lineItems").controller 'LineItemsCtrl', ($scope, $timeout,
     else
       StatusMessage.display 'failure', t "unsaved_changes_error"
 
+  $scope.getOrdersToCancel = (lineItemsToDelete) ->
+    deleteCount = new Map()
+    ordersToCancel = []
+    for item in lineItemsToDelete
+      { order } = item
+      deleteCount.set(
+        order,
+        (deleteCount.get(order) ? 0) + 1)
+      if (order.item_count == deleteCount.get(order))
+        ordersToCancel.push(order)
+    return ordersToCancel
+
+  $scope.cancelOrder = (order) ->
```

Comment:
This method for cancelling orders is used elsewhere in the codebase. However, it doesn't change cancelled order's shipment status (as described in issue #8131: selecting "yes" should cancel the order -> changing the order status to `cancelled` and shipment status to `none`. Is this ok? Or am I missing something?